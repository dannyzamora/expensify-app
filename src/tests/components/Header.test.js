import {shallow} from 'enzyme';
import React from 'react';
import Header from '../../components/Header';

test('should render header correctly',()=> {
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();    
    // const renderer = new ReactShallowRenderer;
    // renderer.render(<Header/>)
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
})