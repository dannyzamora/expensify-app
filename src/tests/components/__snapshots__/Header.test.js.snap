// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
